Remotion/Web/Development.WebTesting.IntegrationTests/WebTreeViewControlObjectTest.cs

```diff
@@ -45,5 +47,59 @@ private WebFormsTestPageObject Start ()
     {
       return Start<WebFormsTestPageObject> ("WebTreeViewTest.aspx");
     }
+
+    [Test]
+    public void OrderedWebTree_Does_Order_Nodes_By_Category ()
+    {
+      var home = Start();
+
+      var treeView = home.WebTreeViews().GetByLocalID ("MyOrderedWebTreeView");
+      var nodeTexts = new List<string>();
+      for (var i = 1; i <= 7; i++)
+      {
+        nodeTexts.Add (treeView.GetNode (i).GetText());
+      }
+
+      var expectedNodeTexts = new[] { "1", "7", "2", "4", "3", "5", "6" };
+      Assert.That (nodeTexts, Is.EqualTo (expectedNodeTexts));
+    }
+
+    [Test]
+    public void UnorderedWebTree_Does_Not_Order_Nodes_By_Category ()
+    {
+      var home = Start();
+
+      var treeView = home.WebTreeViews().GetByLocalID ("My");
+      var nodeTexts = new List<string>();
+      for (var i = 1; i <= 7; i++)
+      {
+        nodeTexts.Add (treeView.GetNode (i).GetText());
+      }
+
+      var expectedNodeTexts = new[] { "1", "2", "3", "4", "5", "6", "7" };
+      Assert.That (nodeTexts, Is.EqualTo (expectedNodeTexts));
+    }
+
+    [Test]
+    public void GetCategory_OnNodeWithCategory_ReturnsCategory ()
+    {
+      var home = Start();
+
+      var treeView = home.WebTreeViews().GetByLocalID ("MyWebTreeViewWithCategories");
+      var treeViewNode = treeView.GetNode (1);
+
+      Assert.That (treeViewNode.GetCategory(), Is.EqualTo ("a category"));
+    }
+
+    [Test]
+    public void GetCategory_OnNodeWithOutCategory_ReturnsNull ()
+    {
+      var home = Start();
+
+      var treeView = home.WebTreeViews().GetByLocalID ("MyWebTreeViewWithoutCategories");
+      var treeViewNode = treeView.GetNode (1);
+
+      Assert.That (treeViewNode.GetCategory(), Is.Null);
```

Comment:
please check, I think we never use nullable values in web testing objects, only empty.

Reply:
In #210 there's also an assertion where the badge text is null, so I think this one should be ok.
https://github.com/re-motion/Framework/pull/210/files#diff-b396ccf22e32989a99628253a95fd25de2e56df21742ae06dc11199a280db38fR54-R63

Reply:
Thanks, that's because I apparently missed the chance to ask this during the review. We should definitely clear this up.